### ISSUE TITLE
[question] Sinatra vs Plug, Style suggestions without explanation

### ISSUE THREAD

**Issue by duksis** [elixir-community/elixir.community#14]
Thanks for this awesome project! :heart: 

But I still have some questions:
1. What is the reasoning behind using sinatra instead of plug?
2. Regarding Styleguide - Suggestions without explanation always make me suspicious, do you think it would be feasible to add an explanation eg. "why?" to each style suggestion?

**Comment by barisbalic** [comment]
@duksis no problem, although I can't really gauge how useful it is so I appreciate any feedback!
1. I think it should actually be static, using a generator like middleman or some other 'thing', but I wasn't sure anyone would visit so I went with what was the quickest solution overall.  Haven't checked traffic in awhile but it's usually a visit or two per day, so I'm not sure whether to repurpose the domain somehow or do something else.
2. Agreed, this is effectively a copy of the https://github.com/niftyn8/elixir_style_guide which might change more frequently, I did ask if @niftyn8 would consider contributing directly to this copy but I didn't get a reply which I take to be a polite declination, and given it's current status I can understand why.
